src/components/InfoBox/InfoBox.tsx

```diff
@@ -1,18 +1,47 @@
-import { ThemeIcon, Tooltip } from '@mantine/core';
+import { useState } from 'react';
+import { ActionIcon, Code, Group, Popover } from '@mantine/core';
 
 import { InformationIcon } from '@/icons/icons';
 
+import { CopyToClipboardButton } from '../CopyToClipboardButton/CopyToClipboardButton';
+
 interface Props {
   text: React.ReactNode;
+  uri?: string;
```

Comment:
Add a comment describing this prop?  Not super clear what it's used for, since info-boxes are completely separate from properties. 

Honestly, I would personally prefer to move this out of the InfoBox component, and maybe make a separate component for the properties that add the URI part. And have the InfoBox know nothing about URIs.  Thoughts? 

I.e, that component would do this: 
```
      <InfoBox
        text={
          <>
            {description}
            <Group pt={'sm'}>
              <Code>Copy URI:</Code>
              <CopyToClipboardButton value={uri} />
            </Group>
          </>
        }
      />
```
Thoughts?

Reply:
Made InfoBox more general, now it accepts children. Lmk what you think

Reply:
👍 Good solution! Makes perfect sense :)